rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/core/RSocketConnectorBuilder.kt

```diff
@@ -41,6 +42,18 @@ public class RSocketConnectorBuilder internal constructor() {
         acceptor = block
     }
 
+    public fun reconnectable(retries: Long) {
+        reconnectPredicate = { _, attempt -> attempt < retries }
```

Comment:
this now feels odd given the config style from the other PR

Reply:
What you can propose as an alternative?
I will add some examples of it's usage like this:
```kotlin
RSocketConnector {
  connectionConfig {
    keepAlive = KeepAlive(10.seconds)
  }
  reconnectable(5)
  //or
  reconnectable { _, attempt -> delay(500); attempt < 10 }
}
```

Reply:
I think get rid of the "="

Reply:
there will be no `=` on user side (as I showed in comment)